tests/docstrings/T10.icry.stdout

```diff
@@ -22,12 +22,32 @@ Checking module T10
   Docstrings on T10::where_at__30_11::N
   Docstrings on T10::F1::N
   Docstrings on T10::M::q
+    :exhaust q
```

Comment:
Relatedly, I wonder if this output should say something like "`:exhaust q (implicit)`" or some such to indicate that this is running due to the implicit behavior of how `:check-docstrings` works, rather than an explicit directive in the docstrings themselves.

Reply:
9cb4dde6